.travis.yml

```diff
@@ -105,6 +105,7 @@ before_script:
 
 script:
   - echo "script start"
+  - source activate pandas-dev
```

Comment:
this looks good, but travis is complaining that the environment doesn't exist. This is likely to be because when doing the `conda env create` in `ci/install_travis.sh` the `-n pandas` is still used, and the environment is not created with the right name `pandas-dev`

also, in line 119 below, the source activate could be removed if it's still active from here (I don't think so), or change the name of the activated environment to `pandas-dev` too